### PR TITLE
feat(content-releases): add delete release action button

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

- Create a ReleaseActionMenu component to handle delete for now and updates in the future
- Add the ReleaseActionMenu to the CMReleaseContainer to remove and entry from a release (delete release action)
- Removes unnecessary properties from the permissions

### Why is it needed?

- To remove an entry from a release

### How to test it?

- Go to the content manager select any draft and publish entry
- Add the entry to a release
- Click the 3 dots button and remove it from the release

Permissions check:
- To see the 3 dots button you need permissions to "Edit" or "Remove an entry from a release"
- To see the Remove entry from a release option in the Menu you need permissions to "Remove an entry from a Release"

NOTE:

The simple menu has some issues coming from the DS. These are highlighted with TODOs in the code. I wasn't able to find all the fixes and open a PR over there in time. I will look into it, open a bug report and/or PR over there before circling back here before or after this is merged to update. I've created CS-465 to address these issues.

### CODE REVIEW

File: packages/core/content-releases/server/src/services/release.ts [hunk 1]
```diff
@@ -108,6 +108,7 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
         return {
           ...release,
           action: {
+            ...actionForEntry,
             type: actionForEntry.type,
```
Comment (Feranchz) [inline packages/core/content-releases/server/src/services/release.ts:112]: If you spread all `actionForEntry` you can remove the type: actionForEntry, right?

Comment (markkaylor) [reply packages/core/content-releases/server/src/services/release.ts:112]: definitely, idk what I was thinking

**Review by simotae14 — CHANGES_REQUESTED** [review]
You just need to fix the unit test, the Permissions and the action work as expected

**Review by simotae14 — APPROVED** [review]
nice work

**Pull request merged**